data-files/samples/sample.User.Any

```diff
@@ -0,0 +1,11 @@
+{ 
+    settingsVersion = 1; 
+    users = ( 
+        { 
+            cmp360 = 20; 
```

Comment:
Would we rather specify `mouseDegPerMillimeter` here? Both work, but I believe `mouseDegPerMillimeter` overrides `cmp360` if specified.

Reply:
Agreed